src/renderer/Renderer.js

```diff
@@ -423,6 +428,14 @@ export default class Renderer {
                 freeTexUnit++;
             }
 
+            if (tile.type === 'line' /* || tile.type === 'polygon' */) {
+                gl.clearDepth(1);
```

Comment:
Trolling myself here, I think that calling `gl.depthRange(0,1)` here would be the same thing to do before using the Z buffer